CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
This is a standard Code of Conduct generated with `npx covgen support@flamelink.io`